deploy/charts/cert-manager/crds/clusterissuers.yaml

```diff
@@ -146,6 +146,7 @@ spec:
                   description: Solvers is a list of challenge solvers that will be
                     used to solve ACME challenges for the matching domains.
                   type: array
+                  minItems: 1
```

Comment:
We don't *require* a solver is specified on an issuer. It is technically valid to have an issuer without any solvers defined, despite it being fairly useless.